examples/httpserver_cpython.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2024 Ted Timmons
+#
+# SPDX-License-Identifier: Unlicense
+#
+# Simple example showing how to use httpserver in cpython (e.g., on a computer).
+
+import socket
+from adafruit_httpserver import Server, Request, Response
```

Comment:
Every other example has a blank line between general imports and `adafruit_httpserver` imports